src/modules/DocApp/DocApp.tsx

```diff
@@ -30,9 +30,15 @@ const DocApp = ({ markdownContent, docStructure, paths, directory }: DocAppProps
       <ul>
         {keys.map((key) => {
           const finalUrl = assignUrl(key) ?? '';
+
           return (
             <li key={key}>
-              <Link href={finalUrl.endsWith(key) ? finalUrl : ''}>{key}</Link>
+              {/*
+                This will add the url to paths
+                while removing 'src/' from paths
+                to make it accessible
+               */}
+              <Link href={finalUrl + key.slice(3)}>{key}</Link>
```

Comment:
I think you a problem with this logic:  The routes paths stll broken 

Some examples:

http://localhost:3000/docs/github/how-to-start-a-new-branch.md-to-start-a-new-branch.md
http://localhost:3000/docs/prettier/current-rules.mdrent-rules.md
http://localhost:3000/docs/three/audio.mdio.md